embedchain/loaders/json.py

```diff
@@ -49,6 +49,9 @@ def _check_content(content):
                         Check - `https://docs.embedchain.ai/data-sources/json`"
             )
 
+        if len(content) > MAX_CONTENT_LENGTH:
+            raise ValueError(f"The content is too long. Maximum allowed length is {MAX_CONTENT_LENGTH} characters.")
```

Comment:
Actually just realized that this is quite limiting. We don't want to limit our users from loading a large json. Can you please solve this a better way or remove it?

Reply:
@deshraj 
According to your comment, I removed string length limitation. Instead, regular expressions have been modified to mitigate the effects of the ReDoS vulnerability. A string with a length of 30000 took about **20** seconds, but after the modification, the time is **0.003** seconds for a string of the same length.

Reply:
@deshraj 
Thanks for checking it out. To be honest, I am new to this area. 
I am sure I could have formatted it, but could you please check?